modules/hostap/Kconfig

```diff
@@ -292,6 +292,10 @@ config WIFI_NM_WPA_SUPPLICANT_DPP
 	select MBEDTLS_X509_CSR_WRITE_C
 	select MBEDTLS_X509_CSR_PARSE_C
 
+config WIFI_NM_WPA_SUPPLICAT_11AX
+	bool "802.11ax support"
+	default y
```

Comment:
Should we depend on `CONFIG_WIFI_NM_WPA_SUPPLICANT_AP`?
The symbol seems to be only used for AP anyway.
```
depends on WIFI_NM_WPA_SUPPLICANT_AP
```

Reply:
Yes, add dependencies.